### PR TITLE
Implemented graphql album query. Added album and photo types

### PR DESCRIPTION
**Pull request by dbenavidesv**
## Summary
- Implemented query for getting albums by id
- Added `Album` and `Photo` type to the graphql schema
- Added error specific to the album API

## Trello Card

https://trello.com/c/Gri9b4rY

### CODE REVIEW

File: app/graphql/albums/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const { queries, schema: queriesSchema } = require('./queries');
+//  const { mutations, schema: mutationSchema } = require('./mutations');
```
Comment (FrancoCoronel) [inline app/graphql/albums/index.js]: Delete this comment code

File: app/models/user.js [hunk 2]
```diff
@@ -1,13 +1,20 @@
 module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define(
-    'user',
+    'User',
```
Comment (FrancoCoronel) [inline app/models/user.js:3]: Why did you modified the name of model?

**Review by FrancoCoronel — CHANGES_REQUESTED** [review]

File: app/models/user.js (hunk 2 above)
Comment (dbenavidesv) [reply app/models/user.js:3]: Because in the bootstrap, when required, the model is renamed anyways, so I figured it would be better if I just name it using UpperCamelCase in the definition.

File: app/graphql/albums/index.js (hunk 1 above)
Comment (dbenavidesv) [reply app/graphql/albums/index.js]: My bad.

File: app/models/user.js (hunk 2 above)
Comment (FrancoCoronel) [reply app/models/user.js:3]: You are right!

File: app/graphql/index.js [hunk 3]
```diff
@@ -1,17 +1,20 @@
-const { makeExecutableSchema } = require('graphql-tools'),
-  types = require('./types'),
-  inputs = require('./inputs'),
-  users = require('./users'),
-  healthCheck = require('./healthCheck');
+const { makeExecutableSchema } = require('graphql-tools');
 
-const typeDefs = [types, inputs, ...users.schemas, ...healthCheck.schemas];
+const types = require('./types');
```
Comment (igna92ts) [inline app/graphql/index.js:3]: why did you change this?

File: app/graphql/users/index.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
-const { queries, schema: queriesSchema } = require('./queries'),
-  { mutations, schema: mutationSchema } = require('./mutations'),
-  { subscriptions, schema: subscriptionsSchema } = require('./subscriptions');
+const { queries, schema: queriesSchema } = require('./queries');
```
Comment (igna92ts) [inline app/graphql/users/index.js:1]: why did you change this too?

File: app/graphql/users/mutations.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
-const { gql } = require('apollo-server'),
-  { userLoggedIn } = require('../events'),
-  { user: User } = require('../../models');
+const { gql } = require('apollo-server');
```
Comment (igna92ts) [inline app/graphql/users/mutations.js:1]: same

File: app/services/albums.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  const options = {
+    method: 'GET',
+    uri: `${albumsApi.endpoint}${albumsApi.routes.albums}/${id}`,
+    json: true
+  };
+  logger.info(`Request to make: ${options.method} ${options.uri}`);
+
+  return request(options)
+    .then(album =>
+      this.getPhotos({ albumId: album.id }).then(photos => ({
```
Comment (igna92ts) [inline app/services/albums.js:17]: use exports rather than this, we try to avoid the usage, otherwise its fine

**Review by igna92ts — APPROVED** [review]

File: app/graphql/index.js (hunk 3 above)
Comment (dbenavidesv) [reply app/graphql/index.js:3]: Amm it's because I like imports to be separated, it doesn't really have any other purpose

File: app/services/albums.js (hunk 6 above)
Comment (dbenavidesv) [reply app/services/albums.js:17]: Ahh ok. Can I change it in the next PR? I have to do similar corrections on that one anyway
